jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/ProbeHandlerTest.java

```diff
@@ -116,14 +110,14 @@ public void getExecProbeWithExecTest() {
 
         probe = probeHandler.getProbe(probeConfig);
         //assertion
-        assertNotNull(probe);
-        assertEquals(5,probe.getInitialDelaySeconds().intValue());
-        assertEquals(5,probe.getTimeoutSeconds().intValue());
-        assertNotNull(probe.getExec());
-        assertEquals(2,probe.getExec().getCommand().size());
-        assertEquals("[cat, /tmp/probe]",probe.getExec().getCommand().toString());
-        assertNull(probe.getHttpGet());
-        assertNull(probe.getTcpSocket());
+        assertThat(probe).isNotNull();
+        assertThat(probe.getInitialDelaySeconds().intValue()).isEqualTo(5);
+        assertThat(probe.getTimeoutSeconds().intValue()).isEqualTo(5);
+        assertThat(probe.getExec()).isNotNull();
+        assertThat(probe.getExec().getCommand().size()).isEqualTo(2);
+        assertThat(probe.getExec().getCommand()).hasToString("[cat, /tmp/probe]");
+        assertThat(probe.getHttpGet()).isNull();
```

Comment:
Please chain assertions here. Try to do assertion using one `assertThat` statement like done in other places.